hibernate-core/src/main/java/org/hibernate/exception/VersionValueException.java

```diff
@@ -0,0 +1,9 @@
+package org.hibernate.exception;
```

Comment:
So this is the wrong package for this exception. (Check the package-level javadoc.)

On the other hand, I'm not sure that this particular condition really deserves it's whole own exception type.

I would consider just using `PropertyValueException` for this; the semantics seem to match pretty closely.

Even if you think a dedicated exception type _is_ worthwhile, I would make `VersionValueException` extend `PropertyValueException`.

Reply:
I'm going to remove `VersionValueException` and use as you suggest a `PropertyValueException`